core/src/main/scala/ch.epfl.scala.index.model/DataPaths.scala

```diff
@@ -1,11 +1,9 @@
-package ch.epfl.scala.index
-package data
+package ch.epfl.scala.index.model
```

Comment:
Why did you move it to `model`? In the future, I would expect it to be goes to `infra`?

Reply:
make sense, it should be next to ESRepo.